pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires-poetry = ">=2.0.0"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-ops = "^2.23.1"
```

Comment:
commit 5870ee71c9f6a091772ed8ddb0d92903ca2d8adf (tag: rev930, tag: rev929)
Author: Dragomir Penev
Date:   Wed Oct 8 20:09:05 2025 +0300

    DPE-8470 Temporary revert to Ops 2 (#1208)
    
    Issue:
    
    Some tests start failing ops v3 (especially self-healing). It requires investigation,
    but 14/stable is long waited for PS6 fixes.
    
    Solution:
    
    Temporary reverting ops to previous version 2 to release the next 14/stable